.new_docs/en/build_and_install/jetson.md

```diff
@@ -1,13 +1,13 @@
-[English](../../en/build_and_install/jetson.md) | 简体中文
 
-# Jetson部署库编译
+# How to Build Jetson Deployment Library
 
-FastDeploy当前在Jetson仅支持ONNX Runtime CPU和TensorRT GPU两种后端推理
+FastDeploy currently supports only ONNX Runtime CPU and TensorRT GPU backend inference in Jetson
```

Comment:
FastDeploy supports CPU inference with ONNX Runtime and GPU inference with Nvidia TensorRT on Nvidia Jetson platform